src/main/java/org/folio/config/ConfigurationsClient.java

```diff
@@ -91,13 +101,13 @@
 
     Promise<SamlConfiguration> result = Promise.promise();
 
-    List<Future> futures = entries.entrySet().stream()
+    List<Future> futures = entries.entrySet().stream() //NOSONAR
```

Comment:
Suppressing sonar warnings is allowed, however, the smallest suppression possible should be chosen.
`NOSONAR` suppresses all warnings and will suppress important warnings that may arise in future.
Use `@SuppressWarnings("squid:S2078")` (replace the number with the number reported by sonar).

In addition add a comment to the suppression describing which warning is suppressed. Neither `NOSONAR` nor `squid:S2078` give this information. The code reviewers need this information to review the suppression.

Reply:
point taken.  You're 100% right